dockers/broad/zcall/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM python:3.9-alpine
+# Zcall needs python2
+FROM python:2.7.18-alpine3.11
```

Comment:
Oh No! more Python 2 :(

Reply:
Zcall is all python2 unfortunately....converting to 3 just breaks everything

Reply:
It is what it is... still sad though